documentation/src/docs/metrics.md

```diff
@@ -7,46 +7,63 @@ You may obtain a copy of the License at
 
     http://www.apache.org/licenses/LICENSE-2.0
 -->
-In Pravega Metrics Framework, we use [Dropwizard Metrics](https://metrics.dropwizard.io/3.1.0/apidocs) as the underlying library, and provide our own API to make it easier to use.
+In Pravega Metrics Framework, we use [Micrometer Metrics](https://micrometer.io/docs) as the underlying library, and provide our own API to make it easier to use.
 # 1. Metrics interfaces and examples usage
-There are four basic interfaces: StatsProvider, StatsLogger (short for Statistics Logger), OpStatsLogger (short for Operation Statistics Logger, and it is included in StatsLogger) and Dynamic Logger.
-StatsProvider provides us the whole Metric service; StatsLogger is the place at which we register and get required Metrics ([Counter](https://metrics.dropwizard.io/3.1.0/manual/core/#counters)/[Gauge](https://metrics.dropwizard.io/3.1.0/manual/core/#gauges)/[Timer](http://metrics.dropwizard.io/3.1.0/manual/core/#timers)/[Histograms](https://metrics.dropwizard.io/3.1.0/manual/core/#histograms)); while OpStatsLogger is a sub-metric for complex ones (Timer/Histograms).
+There are four basic interfaces: StatsProvider, StatsLogger (short for Statistics Logger), OpStatsLogger (short for Operation Statistics Logger, 
+and it is included in StatsLogger) and Dynamic Logger.
+StatsProvider provides us the whole Metric service;
+StatsLogger is the place at which we register and get required Metrics 
+([Counter](https://micrometer.io/docs/concepts#_counters)/
+[Gauge](https://micrometer.io/docs/concepts#_gauges)/
+[Timer](https://micrometer.io/docs/concepts#_timers)/
+[Distribution Summary](https://micrometer.io/docs/concepts#_distribution_summaries)); 
+while OpStatsLogger is a sub-metric for complex ones (Timer/Distribution Summary).
 ## 1.1. Metrics Service Provider — Interface StatsProvider
-The starting point of Pravega Metric framework is the StatsProvider interface, it provides start and stop method for Metric service. Regarding the reporters, currently we have support for CSV reporter and StatsD reporter.
+The starting point of Pravega Metric framework is the StatsProvider interface, it provides start and stop method for Metric service.
+It also provides startWithoutExporting() for unit testing purpose which only store metrics in memory without exporting them
+to outside destinations. 
+Currently we have support StatsD and InfluxDB registries.
 ```java
 public interface StatsProvider {
```

Comment:
Can you please remove this code and add a link to the `StatsProvider` interfaces in the codebase?

Reply:
Though I'm not the original author of the document, my understanding is that the interface is provided here so readers don't have to leave the doc (to get interface definition) when they reach the bullet points following the interface.